problem_one.rb

```diff
@@ -0,0 +1,8 @@
+# Given an integer array of size N, compute the sum of all even numbers in this array.
+
+class ProblemOne
```

Comment:
Do you need classes for this? What is the need for a class, that you'll then have to instantiate just to call a method on it?

Reply:
I actually asked myself the same question! Most of the example tests found were done on methods inside classes, which led me to seeing this as a requirement, although instantiating methods in those three cases seemed somewhat redundant.

Further research proved classes can be avoided in my case using `describe '#method_name'`. 
Will refactor.